common/stacks/src/main/scala/weco/api/stacks/services/SierraService.scala

```diff
@@ -33,6 +34,16 @@ class SierraService(
 )(implicit ec: ExecutionContext)
     extends Logging {
 
+  def getAccessCondition(sourceIdentifier: SourceIdentifier)
+    : Future[Either[SierraItemLookupError, Option[AccessCondition]]] = {
+    val itemNumber = SierraItemIdentifier.fromSourceIdentifier(sourceIdentifier)
+
+    sierraSource.lookupItem(itemNumber).map {
+      case Right(item) => Right(item.getAccessCondition(itemNumber))
+      case Left(err)   => Left(err)
+    }
+  }
```

Comment:
Do you need the pattern match, or can this be a straight map?

```suggestion
    sierraSource.lookupItem(itemNumber)
      .map { item) =>
        Right(item.getAccessCondition(itemNumber))
      }
```

Reply:
Oh dear, I forgot eithers were right biased!